active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,591 @@
+# Teaching tip
+
+## Description
+
+A teaching tip is a content-rich nonmodal flyout that provides contextual information. A teaching tip looks like a flyout, except that it may target a UI element with a pointer or be persistent and require the user to manually dismiss it. A teaching tip is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
+
+**Important APIs:** [TeachingTip class]() [Link TODO]
+
+A teaching tip may also be light-dismiss enabled or used without a pointer indicating reference to any specific UI element. 
+
+
+### Is this the right control? 
+
+Use a **TeachingTip** control to inform users about new and important updates and features, focus users on a component to advise them about its options or state, remind users of nonessential options that would improve their experience, or teach users how a task should be completed. 
+
+Because a teaching tip is transient, it would not be the recommended control for notifying users about important errors and statuses, prompting users for event handling, nor displaying a list of items or events that have been handled. 
```

Comment:
> prompting users for event handling, nor displaying a list of items or events that have been handled.

I'm not sure what this means, sounds too jargony

Reply:
```suggestion
Because teaching tip is transient, it would not be the recommended control for prompting users about errors or important statuses.
```

Reply:
Thanks, @YuliKl! Does this feel like a step in the right direction?

Reply:
Sounds good. Maybe "important status changes"?